test_pull_requests

```diff
@@ -862,8 +1034,8 @@ popd
       repo_to_pull_request.each do |repo, pull_request|
         pull_request_comments = get_comments(pull_request['number'], repo)
         test_prefix_comment = get_comment_with_prefix(pull_request['number'], repo, settings['test_prefix'], pull_request_comments)
-        if !test_prefix_comment || !(test_prefix_comment['body'] =~ /^#{settings['test_prefix']} Waiting/)
-          being_queued_comment = "#{settings['test_prefix']} Waiting: Determining build queue position"
+        if !test_prefix_comment || !(test_prefix_comment['body'] =~ /^#{settings['test_prefix']} #{CONTENT_WAITING}/)
```

Comment:
I thought it was a bit overkill. We know what we're looking for here, we don't need to spend cycles extracting build urls and whatnot. This isn't the hill I want to die on though, so if you feel strongly on this lemme know and I'll change it.